### DISCUSSION TITLE
Import/Export bookmarks to a backup file

### DISCUSSION THREAD

**Discussion by Bardock88** [mozilla-mobile/fenix#24725] [Ideas and Feature Requests]
Will there be any chance to have the bookmark import/export function, via a handy backup file saved on Android phone in the next stable version of Firefox?

**Comment by cadeyrn** [comment]
There is no chance that it will be in the "next stable version of Firefox" (Firefox 100) because it would have to already exist in Firefox Nightly and Beta and that's not the case. Such a feature was already suggested, you can follow #417.

**Reply by Bardock88** [reply]
Ah ok, it's just that I thought there would be time for this feature to reach the next stable version 😅, of course going through NIGHTLY and later BETA.
I will look at this topic, but I say: this feature would help a lot, without a doubt, all browser users.
And thank you very much for answering me 👍.
